cli/loader_lib.c

```diff
@@ -527,8 +527,13 @@ __attribute__((constructor)) void jl_load_libjulia_internal(void) {
     }
     void *fptr = lookup_symbol(RTLD_DEFAULT, "jl_get_pgcstack_static");
     void *(*key)(void) = lookup_symbol(RTLD_DEFAULT, "jl_pgcstack_addr_static");
-    if (fptr != NULL && key != NULL)
-        jl_pgcstack_setkey(fptr, key);
+    _Atomic char *semaphore = lookup_symbol(RTLD_DEFAULT, "jl_pgcstack_static_semaphore");
```

Comment:
```suggestion
    _Atomic(char) *semaphore = lookup_symbol(RTLD_DEFAULT, "jl_pgcstack_static_semaphore");
```

(apparently on Windows, the `_Atomic x` syntax means something slightly different and is not correctly implemented)